references/cli/delete.go

```diff
@@ -67,29 +67,36 @@ func NewDeleteCommand(c common2.Args, ioStreams cmdutil.IOStreams) *cobra.Comman
 		if err != nil {
 			return err
 		}
+		wait, err := cmd.Flags().GetBool("wait")
+		if err != nil {
+			return err
+		}
+		o.Wait = wait
+		force, err := cmd.Flags().GetBool("force")
+		if err != nil {
+			return err
+		}
+		o.ForceDelete = force
+
 		if svcname == "" {
-			ioStreams.Infof("Deleting Application \"%s\"\n", o.AppName)
-			info, err := o.DeleteApp()
-			if err != nil {
-				if apierrors.IsNotFound(err) {
-					ioStreams.Info("Already deleted")
-					return nil
-				}
+			ioStreams.Infof(fmt.Sprintf("Deleting Application \"%s\"\n", o.AppName))
+			if err = o.DeleteApp(ioStreams); err != nil {
 				return err
 			}
-			ioStreams.Info(info)
+			ioStreams.Info(green.Sprintf("app \"%s\" deleted from namespace \"%s\"", o.AppName, o.Namespace))
 		} else {
-			ioStreams.Infof("Deleting Service %s from Application \"%s\"\n", svcname, o.AppName)
+			ioStreams.Infof(fmt.Sprintf("Deleting Service %s from Application \"%s\"\n", svcname, o.AppName))
 			o.CompName = svcname
-			message, err := o.DeleteComponent(ioStreams)
-			if err != nil {
+			if err = o.DeleteComponent(ioStreams); err != nil {
 				return err
 			}
-			ioStreams.Info(message)
+			ioStreams.Info(green.Sprintf("component \"%s\" deleted from \"%s\"", o.CompName, o.AppName))
 		}
 		return nil
 	}
 	cmd.PersistentFlags().StringP(Service, "", "", "delete only the specified service in this app")
+	cmd.PersistentFlags().BoolP("wait", "w", false, "wait util the application is deleted completely")
+	cmd.PersistentFlags().BoolP("force", "f", false, "force to delete the application")
```

Comment:
use `cmd.Flags().BoolVarP(&o.Wait,"wait","w",false,"wait until the application is deleted completely")` will save a lot code.

Reply:
fix it